src/uproot/compression.py

```diff
@@ -138,9 +158,21 @@ class _DecompressLZMA:
     _method = b"\x00"
 
     def decompress(self, data: bytes, uncompressed_bytes=None) -> bytes:
-        import lzma
-
-        return lzma.decompress(data)
+        cramjam = uproot.extras.cramjam()
+        lzma = None
+        if hasattr(cramjam, "experimental") and hasattr(cramjam.experimental, "lzma"):
+            lzma = cramjam.experimental.lzma
+        elif hasattr(cramjam, "lzma"):
+            lzma = cramjam.lzma
+        else:
+            raise RuntimeError(
+                "lzma not found in the cramjam package! (requires cramjam >= 2.8.1)"
+            )
```

Comment:
Slick version (also prioritizing primary/eventual namespace over the experimental namespace):

```suggestion
        lzma = (
            getattr(cramjam, "lzma", None)
            or getattr(getattr(cramjam, "experimental", None), "lzma", None)
        )
        if lzma is None:
            raise RuntimeError(
                "lzma not found in the cramjam package! (requires cramjam >= 2.8.1)"
            )
```